ui/settings.glade

```diff
@@ -3923,6 +3930,71 @@ If available select &lt;small&gt;&lt;tt&gt;pulse&lt;/tt&gt;&lt;/small&gt; as inp
                                 <property name="position">2</property>
                               </packing>
                             </child>
+                            <child>
+                              <object class="GtkFrame" id="sid1">
+                                <property name="visible">True</property>
+                                <property name="can_focus">False</property>
+                                <property name="label_xalign">0.0099999997764825821</property>
+                                <child>
+                                  <object class="GtkAlignment" id="sid2">
+                                    <property name="visible">True</property>
+                                    <property name="can_focus">False</property>
+                                    <property name="bottom_padding">8</property>
+                                    <property name="left_padding">12</property>
+                                    <property name="right_padding">12</property>
+                                    <child>
+                                      <object class="GtkGrid" id="sid3">
+                                        <property name="visible">True</property>
+                                        <property name="can_focus">False</property>
+                                        <property name="row_spacing">10</property>
+                                        <property name="column_spacing">10</property>
+                                        <child>
+                                          <object class="GtkLabel" id="label3">
+                                            <property name="visible">True</property>
+                                            <property name="can_focus">False</property>
+                                            <property name="label" translatable="yes">Default Seek Time</property>
```

Comment:
I'm not really happy with this label. Better would be `Seeking offset in seconds` or along these lines.

Reply:
Am I right that this only modifies the offset applied when pressing the seek buttons?

Reply:
Indeed, that option only modifies the offset applied when pressing the seek buttons.
I thought I'd be more appropriate to keep it separate from the actual track seeking when clicking on a stroke. Maybe I could add another configuration setting for that, but as of now I'm quite happy with it